### PR TITLE
fix some error with OffloaderV1

### PR DESCRIPTION
**Pull request by Hide-on-bushsh**
## Motivation
when use deepseek model and set --cpu-offload-gb in feature offloaderV1, it doesn't work with npu and gpu. Then I fix some error to support those features.

## Modifications

Comparison with Other Models, DeepSeek has a special design, Some tensors are not nn.Parameter and will not be covered by the device processing logic of OffloaderV1.
image
so these issues have been fixed：
1- The correction_bias device mismatch issue in TopK was resolved by adding device processing code to the TopK.forward_npu function.
2- functional_call does not support duplicate keys. The forward method of OffloaderV1 was modified to use named_parameters() and the seen_params set to avoid repeated processing of bound parameters.
3- The w_kc and w_vc device mismatch issue in MLA was resolved by adding device processing code to the forward_mla_prepare_npu function.

For this configuration:
python -m sglang.launch_server \
--model-path /home/weights/deepseekv3-lite-base-latest \
--host 127.0.0.1 \
--port 8080 \
--attention-backend ascend \
--mem-fraction-static 0.9 \
--base-gpu-id 14 \
--tp 1 \
--dp 1 \
--cpu-offload-gb 1 \
--disable-cuda-graph \

curl --location 'http://127.0.0.1:8080/generate' --header 'Content-Type:application/json' --data '{"text": "The captial of France is", "sampling_params": {"temperature": 0, "max_new_tokens": 20}}

**Before(npu)**
image

After the first modification(npu and gpu):
image
After the second modification:
npu
image
gpu
image

After the third modification，it can perform inference properly without any error reported.
NPU
image
GPU
image

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review Process

1. Ping Merge Oncalls to start the PR flow. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - `/tag-run-ci-label`, `/rerun-failed-ci`, `/tag-and-rerun-ci`
4. After green CI and required approvals, ask Merge Oncalls to merge.

---
### CI States

Latest PR Test (Base): :x: [Run #28697970075](https://github.com/sgl-project/sglang/actions/runs/28697970075)
Latest PR Test (Extra): :x: [Run #28697970052](https://github.com/sgl-project/sglang/actions/runs/28697970052)

### CODE REVIEW

File: python/sglang/srt/utils/offloader.py [hunk 1]
```diff
@@ -110,7 +110,13 @@ def maybe_offload_to_cpu(self, module: torch.nn.Module) -> torch.nn.Module:
         # offload parameters to CPU
         # use pin_memory if possible, which helps cudagraph capture speed
         offloaded_parameters = False
-        for p in module.parameters():
+        for name, p in module.named_parameters():
+            # Skip MoE weights (w13_weihgt, w2_weight) for NPU devices
+            # These weights need to maintain FRACTAL_NZ format
+            if str(device).startswith("npu"):
```
Comment (husf1130) [inline python/sglang/srt/utils/offloader.py]: why use `startwith`, instead of `str(device) == "npu"`

Comment (Hide-on-bushsh) [reply python/sglang/srt/utils/offloader.py]: The output of str(device) like this 'npu:15'
